lib/Sema/DebuggerTestingTransform.cpp

```diff
@@ -74,12 +74,17 @@ class DebuggerTestingTransform : public ASTWalker {
   bool walkToDeclPre(Decl *D) override {
     pushLocalDeclContext(D);
 
+    if (D->isImplicit())
```

Comment:
Why? Can you add a comment?

Reply:
Implicit decls are created by the compiler, and are usually not under the direct control of users.